src/lib/classes/universal-profile.spec.ts

```diff
@@ -586,7 +662,9 @@ describe('UniversalProfile', () => {
           signers[0]
         );
 
-        const data = await universalProfile.getData([LSP3_UP_KEYS.UNIVERSAL_RECEIVER_DELEGATE_KEY]);
+        const data = await universalProfile['getData(bytes32[])']([
```

Comment:
maybe `data` is not a perfect variable name.
What about `universalReceiverDelegateAddress` ?